.circleci/config.yml

```diff
@@ -1,9 +1,12 @@
 version: 2.1
+orbs:
+  secrethub: secrethub/cli@1.0.0
 jobs:
   build:
     docker:
       - image: circleci/golang:1.13
     steps:
+      - secrethub/install
```

Comment:
Do you think we should add a comment explaining why we had to add this in?

Reply:
Not necessary IMO.